src/components/containers/victory-brush-container.js

```diff
@@ -116,8 +115,7 @@ export default class VictoryBrushContainer extends VictoryContainer {
   // Overrides method in VictoryContainer
   getChildren(props) {
     const children = React.Children.toArray(props.children);
-    const components = [...children, this.getRect(props)];
-    return components.map((component, i) => {
+    return map([...children, this.getRect(props)], (component, i) => {
```

Comment:
I'm spreading arrays all over the place... Let's decide later if we want to make that systematic change.